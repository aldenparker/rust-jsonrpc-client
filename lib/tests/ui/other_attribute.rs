#[jsonrpc_client::api(foo = "bar")]
pub trait Math {
    async fn subtract(&self, subtrahend: i64, minuend: i64) -> i64;
}

fn main() {}
