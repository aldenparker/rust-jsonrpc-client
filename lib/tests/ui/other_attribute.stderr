error: invalid configuration attribute, currently only `version` is supported
 --> $DIR/other_attribute.rs:1:23
  |
1 | #[jsonrpc_client::api(foo = "bar")]
  |                       ^^^
